apps/app/pages/_app.js

```diff
@@ -1,6 +1,10 @@
 import Head from 'next/head';
+import '../../../packages/tailwind-constructor/output/core.css';
```

Comment:
this is not needed as we're already importing tailwind.css in this file.

Reply:
Earlier, I encountered an error indicating that CSS should be included globally. This occurred when I didn't use the Tailwind class "bg-LightBlue-lightblue-100" in this manner. In the latest commit, I removed the comments and made minor changes. Thanks!